Axeptio documentation (#1033)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/axeptio.mdx

1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **Axeptio**.
2. Assign a name to the destination and click **Continue**.

## Connection settings

To successfully configure Axeptio as a destination, you will need to configure the following settings:

- **Client ID**: Enter your Axeptio client ID. For more information on obtaining the client ID, refer to the <Link to="#faq">FAQ</Link> section below.
- **Toggle it on to send data through callback**: Enable this setting to allow RudderStack to track and record the callback events from the Axeptio SDK as `track` events.
- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Axeptio. For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
- **Use device mode to send events**: Enable this setting to send your events to Axeptio via the <Link to="https://www.rudderstack.com/docs/destinations/rudderstack-connection-modes/#device-mode">web device mode</Link>.

<div class="infoBlock">

> You can disable this setting if you only want to load the Axeptio SDK natively and not send any events.

Reviewer: ```suggestion
You can keep it disabled if you only want to load the Axeptio SDK natively and not send any events.
```

</div>

- **OneTrust Cookie Categories**: This setting lets you associate <Link to="/sources/sdks/rudderstack-javascript-sdk/consent-managers/onetrust/">OneTrust</Link> cookie consent groups to Axeptio.

## Tracking callback events

The Axeptio SDK triggers events whenever a user interacts with the cookie/consent widgets integrated with your website.

> By enabling the **Toggle it on to send data through callback** setting in the RudderStack dashboard, you can track and record the interaction events as <Link to="/event-spec/standard-events/track/">`track`</Link> events. 

Reviewer 2: We can make it like this.
```suggestion
By enabling the **Toggle it on to send data through callback** setting in the RudderStack dashboard, RudderStack tracks and records the interaction events as <Link to="/event-spec/standard-events/track/">`track`</Link> events. You can then view and analyze these events in the other tools (connected to the same source in RudderStack).
```

RudderStack maps the name and the payload of the triggered interaction event to the `track` event name and properties respectively.

## FAQ

### Where can I find the Axeptio client ID?

To get the client ID associated with your Axeptio project, follow these steps:

1. Log into your <a href="https://admin.axeptio.eu/">Axeptio dashboard</a>.
2. Click the settings icon on your project. You should be able to see the **Project ID** listed here, as shown:

<img src="../../assets/event-stream-destinations/axeptio-client-id.png" alt="Axeptio client ID" />

You can also find the client ID in the code snippet to integrate into your website. Click the settings icon on your project and go to **Integration** to find the client ID, as shown:

<img src="../../assets/event-stream-destinations/axeptio-client-id-2.png" alt="Axeptio client ID" />

<div class="infoBlock">

> The Axeptio client ID is the same as the project ID.

Reviewer: ```suggestion
Axeptio client ID is the same as the project ID.
```
